gkeys/gkeys/actions.py

```diff
@@ -339,7 +344,27 @@ def checkkey(self, args):
                 _unicode("\n  =============================================="))
             self.logger.debug(_unicode("ACTIONS: checkkey; gkey = %s") % _unicode(gkey))
             for key in gkey.pub_keyid:
-                results[gkey.name] = self.gpg.check_keys(gkey.keydir, key)
+                results_ = self.gpg.list_keys(gkey.keydir, fingerprint=key, colons=True)
+                if gkey.uid:
+                    uid = gkey.uid[0]
+                else:
+                    uid = "Invalid_email"
+                match = re.findall(r'[\w\.-]+@[\w\.-]+', uid)
+                if match:
+                    uid = match[0]
+                else:
+                    uid = "Invalid_email"
+                for data in results_.status.data:
+                    if data.name in ("PUB", "SUB"):
+                        print (data.expiredate, "expire_date")
+                        if data.expiredate:
+                            time_diff = int(data.expiredate) - time.time()
+                        else:
+                            time_diff = 'INF'
+                        print(time_diff, "time_diff")
```

Comment:
Why are there print()s in here, if they are debug and you want to keep them then use the logger.  Otherwise nuke them, they don't belong in a pull request.